<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>722</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1942</width>
    <height>1167</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LoginWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>431</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(64,66,68,120);

border-radius:8px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>110</y>
      <width>211</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="AccountEdit">
       <property name="styleSheet">
        <string notr="true">background-color : transparent;
border-color: : transparent;
color : rgb(255,255,255);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="PasswdEdit">
       <property name="styleSheet">
        <string notr="true">background-color : transparent;
border-color: : transparent;
color : rgb(255,255,255);</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="styleSheet">
        <string notr="true">QComboBox {
    border: 1px solid white;   
    border-radius: 3px;   
    padding: 1px 18px 1px 3px;   
    color: #fff;
    font: normal normal 15px &quot;Microsoft YaHei&quot;;
    background: transparent;
}
/* 下拉后，整个下拉窗体每项的样式 */
QComboBox QAbstractItemView::item {
    height: 30px;   /* 项的高度（设置pComboBox-&gt;setView(new QListView());后，该项才起作用） */
	background-color: rgba(100,106,100,150);
	color: rgb(64,64,64);
}
QComboBox QAbstractItemView::item:hover{
	background-color: rgba(255,255,191,100);
	color: rgb(64,64,64);
}
/* QComboBox中的垂直滚动条 */
QComboBox QAbstractScrollArea QScrollBar:vertical {
    width: 10px;
    background-color: #d0d2d4;   /* 空白区域的背景色  灰色green */
	border-radius: 3px;   
}


</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Login">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>360</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
                   background-color:rgba(220,220,220,120);
                   border-style:outset;                                                             
                   border-radius:5px;                     
                   font: 75 15pt &quot;华文行楷&quot;;           
                   padding:6px;
                   }

                   QPushButton:pressed{
                   background-color:rgba(100,100,100,180);
                   border-color:rgba(100,100,100,30);
                   border-style:inset;
                   }
                   QPushButton:hover{
                   background-color:rgba(100,255,100,100);
                   border-color:rgba(255,255,255,200);
}
</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Login_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>430</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
                   background-color:transparent;
                   border-style:outset;                                                             
                   border-radius:8px;                     
                   font: 75 10pt &quot;华文行楷&quot;;           
                   padding:6px;
				   color:rgba(255,255,255,255);
                   }

                   QPushButton:pressed{
                   color:rgba(100,100,100,30);
                   border-style:inset;
                   }
                   QPushButton:hover{
                   color:rgba(0,0,120,50);
}
</string>
    </property>
    <property name="text">
     <string>还没有账号？现在注册！</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>971</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(64,66,68,120);

border-radius:8px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>951</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(64,66,68,120);

border-radius:8px;

 QScrollBar::handle:vertical 
 { 
 width:8px; 
 background:rgba(0,0,0,50%); 
 border-radius:4px; 
 min-height:80 
 } 
 QScrollBar::handle:vertical:hover 
 { 
 width:8px; 
 background:rgba(0,0,0,70%); 
  border-radius:4px; 
 min-height:80; 
 } 
 QScrollBar::add-line:vertical 
 { 
 height:0px;width:8px; 
 border-image:url(:/images/null/null.png); 
 subcontrol-position:bottom; 
 } 
QScrollBar::sub-line:vertical
{
height:0px;width:8px;
border-image:url(:/images/null/null.png);
subcontrol-position:top;
}
QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical
{
background:transparent;
border-radius:4px;
};



</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#ababab;&quot;&gt;	欢迎使用&lt;/span&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#f8f8f8;&quot;&gt;智慧医疗系统&lt;/span&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#ababab;&quot;&gt;，新用户请注册您的新账号！注册完成后，请重新登录并就诊。祝您生活愉快！&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:10pt; font-weight:600; color:#ababab;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:10pt; font-weight:600; color:#ababab;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#ababab;&quot;&gt;	&amp;quot;你每天都在做很多看起来毫无意义的决定，但某天你的某个决定就能改变你的一生。&amp;quot;&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:10pt; font-weight:600; color:#ababab;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#ababab;&quot;&gt;							——《西雅图不眠夜》&lt;/span&gt;&lt;span style=&quot; font-weight:600; color:#ababab;&quot;&gt;&lt;br /&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>260</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="styleSheet">
        <string notr="true">font: 75 11pt &quot;华文行楷&quot;;
color : rgb(255,255,255);</string>
       </property>
       <property name="text">
        <string>Log Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="LoginAuto">
       <property name="styleSheet">
        <string notr="true">font: 75 11pt &quot;华文行楷&quot;;
color : rgb(255,255,255);</string>
       </property>
       <property name="text">
        <string>Login Automaticly</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="warning">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>310</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="passwdBtn">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>430</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
                   background-color:transparent;
                   border-style:outset;                                                             
                   border-radius:8px;                     
                   font: 75 10pt &quot;华文行楷&quot;;           
                   padding:6px;
				   color:rgba(255,255,255,255);
                   }

                   QPushButton:pressed{
                   color:rgba(100,100,100,30);
                   border-style:inset;
                   }
                   QPushButton:hover{
                   color:rgba(0,0,120,50);
}
</string>
    </property>
    <property name="text">
     <string>忘记密码</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>110</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 75 15pt &quot;华文行楷&quot;;
color : rgb(255,255,255);</string>
       </property>
       <property name="text">
        <string>Account：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">font: 75 15pt &quot;华文行楷&quot;;
color : rgb(255,255,255);</string>
       </property>
       <property name="text">
        <string>Password：</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>label_4</zorder>
   <zorder>label</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>Login</zorder>
   <zorder>Login_2</zorder>
   <zorder>textBrowser</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>warning</zorder>
   <zorder>passwdBtn</zorder>
   <zorder>verticalLayoutWidget_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1007</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>日志</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
